packages/cli/src/ipfs-connection-factory.ts

```diff
@@ -65,6 +70,40 @@ export class IpfsConnectionFactory {
           '/ip4/104.131.131.82/tcp/4001/p2p/QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ',
           '/ip4/104.131.131.82/udp/4001/quic/p2p/QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ',
         ],
+        Peering: {
+          Peers: [
+            {
+              Addrs: [
+                '/dns4/go-ipfs-ceramic-private-mainnet-external.3boxlabs.com/tcp/4011/ws/p2p/QmXALVsXZwPWTUbsT8G6VVzzgTJaAWRUD7FWL5f7d5ubAL',
+              ],
+              ID: 'QmXALVsXZwPWTUbsT8G6VVzzgTJaAWRUD7FWL5f7d5ubAL',
+            },
+            {
+              Addrs: [
+                '/dns4/go-ipfs-ceramic-private-cas-mainnet-external.3boxlabs.com/tcp/4011/ws/p2p/QmUvEKXuorR7YksrVgA7yKGbfjWHuCRisw2cH9iqRVM9P8',
+              ],
+              ID: 'QmUvEKXuorR7YksrVgA7yKGbfjWHuCRisw2cH9iqRVM9P8',
+            },
+            {
```

Comment:
Would you mind taking out the two ELP multiaddrs? These nodes are no longer in service. We might reuse their peer IDs later but the DNS names will change so these entries will be stale regardless.

Reply:
WDYT @stbrody? We _could_ also retain the ELP DNS names so that old Ceramic nodes with this list will just work with any future nodes reusing the ELP multiaddrs, even if they're not "ELP" nodes. That feels a little hacky though.

Reply:
yeah I actually feel like it's nice to leave them in and know we have multiaddrs in our back pocket we can use that will be strongly connected to even old nodes